doc/nrf/config_and_build/dfu/index.rst

```diff
@@ -0,0 +1,49 @@
+.. _app_dfu:
+.. _ug_fw_update:
+
+Device Firmware Updates
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+DFU is...
+
+You can deliver the updated images to the device in two ways:
+
+* Wired, where updates are sent through a wired connection, like UART, or delivered by connecting a flash device.
+* Over-the-air (OTA), where updates are sent through a wireless connection, like Bluetooth® Low Energy.
+
+The DFU process depends on the choice of :ref:`bootloader <app_bootloaders>`.
+Generally, bootloaders support two types of updates:
+
+* Direct updates, where an in-place substitution of the image takes place.
+* Background updates, where the updated image is obtained and stored, but the update is completed later on.
+
+Based on these criteria, the |NCS| offers support for the following DFU alternatives:
+
+.. _app_dfu_support_table:
+
+.. list-table:: DFU methods supported by |NCS|
```

Comment:
In this table, it looks like the alternatives are related:
![image](https://github.com/nrfconnect/sdk-nrf/assets/89006246/c6362099-b170-47be-b6e2-d674a79de88f)
However, they are not.
Maybe we can try two separate tables: one for Sending Updates and one for Receiving updates? 
Then have the alternatives in rows rather than columns?

Reply:
For "Receiving Update", FOTA Download uses DFU Target under the hood.
Also, our cloud libraries such as for nRF Cloud, can automatically call FOTA Download from under their hood as well.
So users will use or not use the libraries depending on what they try to do. Not sure if that is  something we should document or not

Reply:
I've removed the table and used bullets instead.